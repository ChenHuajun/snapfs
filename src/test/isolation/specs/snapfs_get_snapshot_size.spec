teardown {
	select pg_remove_snapshot( ( select recent_snapshot from pg_control_snapshot() ) );
	DROP TABLE IF EXISTS foo;
}

session "s1"
step "s1_mk_sn" {
	select 1 as "make_snapshot" from pg_make_snapshot();
	select pg_get_snapshot_size( ( select recent_snapshot from pg_control_snapshot() ) );
	create table foo as select 'test_data';
	select pg_get_snapshot_size( ( select recent_snapshot from pg_control_snapshot() ) ) > 0 as size_not_zero;
}

permutation "s1_mk_sn"
