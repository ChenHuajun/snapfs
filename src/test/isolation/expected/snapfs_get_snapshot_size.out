Parsed test spec with 1 sessions

starting permutation: s1_mk_sn
step s1_mk_sn: 
	select 1 as "make_snapshot" from pg_make_snapshot();
	select pg_get_snapshot_size( ( select recent_snapshot from pg_control_snapshot() ) );
	create table foo as select 'test_data';
	select pg_get_snapshot_size( ( select recent_snapshot from pg_control_snapshot() ) ) > 0 as size_not_zero;

make_snapshot  

1              
pg_get_snapshot_size

0              
size_not_zero  

t              
pg_remove_snapshot

               
